docs/docsite/rst/release_and_maintenance.rst

```diff
@@ -0,0 +1,143 @@
+Release and maintenance
+=======================
+
+.. contents:: Topics
+   :local:
+
+.. _schedule:
+
+Release cycle
+`````````````
+
+Ansible is developed and released on a flexible 4 months release cycle.
+This cycle can be extended in order to allow for larger changes to be properly
+implemented and tested before a new release is made available.
+
+Ansible supports the two most recent major stable releases.
+For more information, read about the
+`development and stable version maintenance workflow`_.
+
+If you are using a release of Ansible that is no longer supported, we strongly
+encourage you to upgrade as soon as possible in order to benefit from the
+latest features and security fixes.
+
+Older unsupported versions of Ansible can contain unfixed security
+vulnerabilities (*CVE*).
+
+You can refer to the `porting guide`_ for tips on updating your Ansible
+playbooks to run on newer versions.
+
+.. _porting guide: https://docs.ansible.com/ansible/porting_guide_2.0.html
+
+Release status
+``````````````
+
++-----------------+----------------------------+----------------------------------------+
+| Ansible release | Latest version             | Status                                 |
++=================+============================+========================================+
+| devel           | `2.4`_ (unreleased, trunk) | In development                         |
++-----------------+----------------------------+----------------------------------------+
+| 2.3             | `2.3.1`_ (2017-06-01)      | Supported (bug **and** security fixes) |
++-----------------+----------------------------+----------------------------------------+
+| 2.2             | `2.2.3`_ (2017-05-09)      | Supported (**only** security fixes)    |
++-----------------+----------------------------+----------------------------------------+
+| 2.1             | `2.1.6`_ (2017-06-01)      | Unsupported (end of life)              |
++-----------------+----------------------------+----------------------------------------+
+| 2.0             | `2.0.2`_ (2016-04-19)      | Unsupported (end of life)              |
++-----------------+----------------------------+----------------------------------------+
```

Comment:
I personally wouldn't have a table contain anything less than 2.0.  If anything, 1.9 should be last in this table. 

Just a nitpick, but not a huge one.